src/index.html

```diff
@@ -11,7 +11,186 @@
+          </label>
+        </div>
+        <div class="form-field">
+          <label>
+            Full date of birth:
+            <input
+            type="date"
+            name="birthdate"
+            >
+          </label>
+        </div>
+        <div>
+          <label>
+            I accept the term of the agreement:
+            <input
+            type="checkbox"
+            name="terms"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>Registrarion</legend>
+
+        <div class="form-field">
+          <label>
+            E-mail:
+            <input
+            type="email"
+            name="email"
+            autocomplete="off"
+            placeholder="email@example.com"
+            >
+          </label>
+        </div>
+        <div>
+          <label>
+            Password:
+            <input
+            type="password"
+            minlength="8"
+            maxlength="32"
+            name="password"
+            autocomplete="off"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          <label>
+            Do you love cats?
+            <input
+            type="radio"
+            name="love-cats"
+            value="y"
+            checked
+            >
+            Yes
+          </label>
+          <label>
+            <input
+            type="radio"
+            name="love-dogs"
+            value="n"
+            >
+            No
+          </label>
+        </div>
+        <div class="form-field">
+          <label>
+            What is your favorite color?
+            <input
+            type="color"
+            name="favcolor"
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label>
+            What time do you go to bed?
+            <input
+            type="time"
+            name="sleep-time"
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label>
+            What are your favorite brand of cars?
+            <select
+            name="car"
+            required
+            multiple
+            >
+              <option value="BMW">BMV</option>
+              <option value="Audi">Audi</option>
+              <option value="KIA">Lada</option>
+            </select>
+          </label>
+        </div>
+        <div>
+          <label>
+            How do you rate our work?
+            <input
+            type="range"
+            name="rating"
+            value="0">
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="group">
+        <legend>Additional info:</legend>
+        <div class="form-field">
+          <label>
+            Comments:
+            <textarea
+            name="comments"
+            cols="15"
+            rows="2"
+            >
```

Comment:
recheck [indentation](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md#:~:text=%5BCODE%20STYLE%5D%20%2D%20If%20the%20HTML%2Delement%20has%20long%20attribute%20values%20or%20number%20of%20attributes%20is%20more%20than%202%20%2D%20start%20each%20one%2C%20including%20the%20first%2C%20on%20the%20new%20line%20with%202%2Dspace%20indentation%20related%20to%20tag.%20Tag%E2%80%99s%20closing%20bracket%20should%20be%20on%20the%20same%20level%20as%20opening%20one.)

```suggestion
          <label>
              Comments:
            <textarea
              name="comments"
              cols="15"
              rows="2"
            >
```